cli/cmd/auth.go

```diff
@@ -32,6 +36,26 @@ More precisely, the Keptn CLI stores the endpoint and API token using *pass* in
 	Example:      `keptn auth --endpoint=https://api.keptn.MY.DOMAIN.COM --api-token=abcd-0123-wxyz-7890`,
 	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
+
+		var err error
+		if *exportConfig {
+			exportEndPoint, exportAPIToken, err = credentialmanager.NewCredentialManager().GetCreds()
+			if err != nil {
+				return err
+			}
+			fmt.Println("Endpoint = ", exportEndPoint.String())
+			fmt.Println("API Token = ", exportAPIToken)
+			return nil
+		}
+
+		if *endPoint == "" && *apiToken == "" {
+			return errors.New("required flag(s) \"api-token\", \"endpoint\" not set")
+		} else if *endPoint == "" {
+			return errors.New("required flag \"endpoint\" not set")
+		} else if *apiToken == "" {
+			return errors.New("required flag \"api-token\" not set")
+		}
+
```

Comment:
We needed a similar check in `keptn configure bridge [--output]`. 

https://github.com/keptn/keptn/blob/7d5fb6e6174a13c7fdfe45650266d18a1fab6780/cli/cmd/configure_bridge.go#L162-L165

Can you refactor the `keptn auth` command such that it also uses a `PreRun` check?

https://github.com/keptn/keptn/blob/7d5fb6e6174a13c7fdfe45650266d18a1fab6780/cli/cmd/configure_bridge.go#L43-L45

Reply:
Sure, I thought to use `PreRun` check, will refactor